mapie/estimator/regressor.py

```diff
@@ -497,13 +454,44 @@ def fit(
                     )
                     for train_index, _ in cv.split(X, y, groups)
                 )
-                # In split-CP, we keep only the model fitted on train dataset
-                if self.use_split_method_:
-                    single_estimator_ = estimators_[0]
 
-        self.single_estimator_ = single_estimator_
-        self.estimators_ = estimators_
+        self.estimators_ = estimators
+
+        return self
+
+    def fit(
```

Comment:
I prefered `fit_single_estimator`: the purpose of the function is clearer

Reply:
Done!